tests/testthat/test-osmodel.R

```diff
@@ -0,0 +1,32 @@
+
+test_that("parametrize filled all the gaps", {
```

Comment:
Would also be good to add some tests to show the loglogistic model is picking up the code as expected.

Reply:
https://github.com/Genentech/jmpost/blob/0123fd37eb9c92d64ab2b941767822157607c02b/tests/testthat/test-osmodel.R#L72-L74